dspace-api/src/main/java/org/dspace/app/bulkedit/MetadataExportFilteredItemsReportCliScriptConfiguration.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+
+package org.dspace.app.bulkedit;
+
+import org.apache.commons.cli.Options;
+
+/**
+ * This is the CLI version of the {@link MetadataExportFilteredItemsReportScriptConfiguration} class that handles the
+ * configuration for the {@link MetadataExportFilteredItemsReportCli} script
+ *
+ * @author Jean-François Morin (Université Laval)
+ */
+public class MetadataExportFilteredItemsReportCliScriptConfiguration
+    extends MetadataExportFilteredItemsReportScriptConfiguration<MetadataExportFilteredItemsReportCli> {
+
+    @Override
+    public Options getOptions() {
+        Options options = super.getOptions();
+        options.addOption("n", "filename", true, "the filename to export to (default: filtered-items-export.csv)");
```

Comment:
Tiny thing, but I've noticed the default filename is *different* based on whether you run this script from the command-line or from the Admin UI.  From the Admin UI, the script will always be named `metadataExportFilteredItems.csv`.  But, from the command-line, as noted here it will be named `filtered-items-export.csv`.

We probably should set a standard default name. I don't have a strong opinion on which you choose, but it should be consistent.

Reply:
Good point, and you gave me an idea: I switched all occurrences of this default filename to a configurable parameter (in the same `contentreport.cfg` file as the activation switch). With this feature there will be no more inconsistencies.